src/test/java/com/atlassian/db/replica/api/AuroraClusterMock.java

```diff
@@ -52,7 +62,6 @@ public static ResultSet auroraGlobalDbInstanceStatus() {
         SimpleResultSet rs = new SimpleResultSet();
         rs.addColumn("SERVER_ID", Types.VARCHAR, 255, 0);
         rs.addColumn("SESSION_ID", Types.VARCHAR, 255, 0);
-        rs.addRow("apg-global-db-rpo-mammothrw-elephantro-1-n1", "MASTER_SESSION_ID");
```

Comment:
Why this is removed?

Reply:
Before the change, the method returned all database nodes, and the h2 database filtered out MASTER_SESSION_ID. Currently, the whole query is mocked, so this part is no longer needed.